contrib/integration/swap/main.go

```diff
@@ -137,10 +137,21 @@ func main() {
 
 }
 
+func getNextWord(index int) string {
+	// check if index is in the range of words
+	if index < 0 || index >= len(words) {
+		x.Log(errors.Errorf("invalid index: %d", index), "get next word")
```

Comment:
we should crash or something here instead of silently failing

Reply:
Done. PTAL